<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReturnDialog</class>
 <widget class="QDialog" name="ReturnDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="eventName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>eventName</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QScrollArea" name="materials">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>932</width>
        <height>290</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="listLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="5">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Actions</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Code barre</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="barCode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="barCodeCommand">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QGroupBox" name="grpHistory">
        <property name="title">
         <string>Historique</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QUndoView" name="undoView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="midLineWidth">
            <number>1</number>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="movement">
            <enum>QListView::Snap</enum>
           </property>
           <property name="emptyLabel">
            <string>Initial</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Affichage</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="btnArchiveAll">
           <property name="text">
            <string>Archiver tous les éléments</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="btnUnArchiveAll">
           <property name="text">
            <string>Désarchiver tous les éléments</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Afficher l'historique</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLineEdit" name="lnFilter"/>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QCheckBox" name="chkFilter">
           <property name="text">
            <string>Filtre</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="barCodeSelet">
   <property name="text">
    <string>Select bacode </string>
   </property>
   <property name="shortcut">
    <string>*, *, *, $</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>barCodeSelet</sender>
   <signal>triggered()</signal>
   <receiver>barCode</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>927</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>barCode</sender>
   <signal>returnPressed()</signal>
   <receiver>ReturnDialog</receiver>
   <slot>barCodeEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnArchiveAll</sender>
   <signal>clicked()</signal>
   <receiver>ReturnDialog</receiver>
   <slot>archiveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>949</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUnArchiveAll</sender>
   <signal>clicked()</signal>
   <receiver>ReturnDialog</receiver>
   <slot>unarchiveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>952</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>grpHistory</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>barCodeEntered()</slot>
  <slot>archiveAll()</slot>
  <slot>unarchiveAll()</slot>
  <slot>enableFilter(bool)</slot>
  <slot>setFilter(QString)</slot>
 </slots>
</ui>
